error_log($logMessage, 3, __DIR__ . '/../../logs/error.log');[2025-05-05 15:47:03] Login error
<<<<<<< HEAD
[2025-05-10 13:38:12] Login error
Exception: Table 'shopgame_db.login_attempts' doesn't exist
File: C:\xampp\htdocs\Baygorn1\app\model\UserModel.php
Line: 167
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\model\UserModel.php(167): mysqli->prepare('SELECT COUNT(*)...')
#1 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(436): UserModel->getLoginAttempts('bao1')
#2 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(69): AuthController->isLockedOut('bao1')
#3 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#4 {main}
[2025-05-10 13:38:22] Login error
Exception: Table 'shopgame_db.login_attempts' doesn't exist
File: C:\xampp\htdocs\Baygorn1\app\model\UserModel.php
Line: 167
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\model\UserModel.php(167): mysqli->prepare('SELECT COUNT(*)...')
#1 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(436): UserModel->getLoginAttempts('bao1')
#2 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(69): AuthController->isLockedOut('bao1')
#3 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#4 {main}
[2025-05-10 13:39:00] Login error
Exception: Table 'shopgame_db.login_attempts' doesn't exist
File: C:\xampp\htdocs\Baygorn1\app\model\UserModel.php
Line: 167
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\model\UserModel.php(167): mysqli->prepare('SELECT COUNT(*)...')
#1 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(436): UserModel->getLoginAttempts('bao1')
#2 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(69): AuthController->isLockedOut('bao1')
#3 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#4 {main}
[2025-05-10 13:39:24] Login error
Exception: Table 'shopgame_db.login_attempts' doesn't exist
File: C:\xampp\htdocs\Baygorn1\app\model\UserModel.php
Line: 167
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\model\UserModel.php(167): mysqli->prepare('SELECT COUNT(*)...')
#1 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(436): UserModel->getLoginAttempts('user1')
#2 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(69): AuthController->isLockedOut('user1')
#3 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#4 {main}
[2025-05-10 13:39:39] Login error
Exception: Table 'shopgame_db.login_attempts' doesn't exist
File: C:\xampp\htdocs\Baygorn1\app\model\UserModel.php
Line: 167
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\model\UserModel.php(167): mysqli->prepare('SELECT COUNT(*)...')
#1 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(436): UserModel->getLoginAttempts('user1')
#2 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(69): AuthController->isLockedOut('user1')
#3 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#4 {main}
=======
[2025-05-10 13:41:27] Login error
Exception: Invalid CSRF token
File: C:\xampp\htdocs\Baygorn1\app\Controller\BaseController.php
Line: 53
Stack trace:
#0 C:\xampp\htdocs\Baygorn1\app\Controller\AuthController.php(49): BaseController->validateCSRF('8c9d7928755dcd7...')
#1 C:\xampp\htdocs\Baygorn1\index.php(114): AuthController->login()
#2 {main}
>>>>>>> 3f3275d015627aaf1cfe35cdfbc7a223fc84091f
